sight_service/service_root.py

```diff
@@ -191,10 +266,15 @@ def __init__(self):
     self.optimizers = Optimizers()
     logging.debug('SightService::__init__')
 
-
   @rpc_call
   def Test(self, request, context):
-    return service_pb2.TestResponse(val="222")
+    method_name = "Test"
```

Comment:
It looks like your merge brought back the logging code I removed when I added the @rpc_call annotations. 

Also, it looks like line 
 obj.action.CopyFrom(convert_dict_to_proto(dict=next_action))
 is there for testing your proto code. Can you please reset this function to just 
 
return service_pb2.TestResponse(val="222")

?

Reply:
yes , it was for testing the proto , removed the code and utils from this file